### ISSUE TITLE
Blackout button interpolation rfe

### ISSUE THREAD

**Issue by drmbt** [theexperiential/nodeo#4]
Retriggering the Off/On blackout button before the fades interpolation has completed results in a value stuck somewhere in between, which is a dangerous behavior for a blackout button, especially in a live event or stream setting. Perhaps a more desirable behavior would be that a click before the interpolation has completed cuts to the value instead of retriggering in the opposite direction? single click to fade, double to cut, with some sort of indicator alarm to draw attention to a blackout would be a solid addition

**Comment by dylanroscover** [comment]
Definitely. I dig your idea of multiple functions for this (fade/cut) as well as indication alarm. I've noticed the "stuck" bug as well. Will prioritize this for the next update, cheers.

**Comment by dylanroscover** [comment]
@drmbt In the latest version I just pushed, the Blackout button triggered via right click will cut, and double click will cut it as well. Seems to be working a bit better now, let me know what you think.

**Comment by drmbt** [comment]
Very cool. Probably won't have a chance to scope it til after this IIHQ thing Saturday.

Have you implemented any further engine optimizations for the UI or the thumbnail system?

**Comment by dylanroscover** [comment]
UI is now fully separated from content (w00t!), so interacting with it doesn't impact output (NDI) fps. Haven't touched the thumbnail system yet... that's up next.
